src/controller/SetUpCodePairer.cpp

```diff
@@ -243,6 +254,19 @@ CHIP_ERROR SetUpCodePairer::StopConnectOverSoftAP()
     return CHIP_NO_ERROR;
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+CHIP_ERROR SetUpCodePairer::StartDiscoverOverWiFiPAF(SetupPayload & payload)
+{
+    return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE;
```

Comment:
We should consider refactoring the interfaces so that there is a generic DiscoveryTransport API that is implemented by both BLE and PAF that the SetupCodePairer and other parts of the stack can use. It seems rather sad to simply duplicate all the code instead.

Reply:
StartDiscoverOverWiFiPAF() has not been implemented and will do it soon. 
As for refactoring the interface, should it be another task which won't be done in here for now?

Reply:
> StartDiscoverOverWiFiPAF() has not been implemented and will do it soon.

Just to be clear: having this work is a pre-requisite for this landing, because "code-wifi" needs to work for PAF, no?

Reply:
The latest commit has added the implementation that code-wifi can work by using wifipaf to do pairing now. Please review.